### ISSUE TITLE
I see thagt you are still working on this

### ISSUE THREAD

**Issue by jewpaltz** [JordonRob/Exercise_Fitness_App#1]
B"H

You seem prepared to put in work for this class, even though you are behind.
Please reach out to me so that we can come up with a plan for that.

**Comment by JordonRob** [comment]
Yes. I'm going to have to schedule a meeting to see how i can improve and catch up. I have been working daily on the project and have made quite a few changes since the last time you saw it. Thanks for keeping in touch

Sent from Yahoo Mail on Android 
 
  On Sun, Oct 25, 2020 at 10:05 PM, Rabbi Moshe Plotkin<notifications@github.com> wrote:   


B"H

You seem prepared to put in work for this class, even though you are behind.
Please reach out to me so that we can come up with a plan for that.
